### PR TITLE
Render correct bill reasons

### LINKED ISSUE #406: Bill reasons in payment history don't match current bills
See #155 for list

"Overdue item" (correct) vs. "Overdue" (not in the translation list)

Screen Shot 2019-08-01 at 4 06 59 PM

"Damaged item" (correct) vs. "Damage fine" (not in the translation list)

Screen Shot 2019-08-01 at 4 08 59 PM

### PR DESCRIPTION
**Pull request by tallenaz**
Fixes #406

### CODE REVIEW

**Comment by tallenaz** [comment]
For user `super1`,
Before:
Screen Shot 2019-08-01 at 7 15 14 PM
After:
Screen Shot 2019-08-01 at 7 12 24 PM

File: app/models/payment.rb [hunk 1]
```diff
@@ -32,7 +32,14 @@ def key
   end
 
   def bill_description
-    record['billReasonDescription']
+    case record['billReasonDescription']
```
Comment (mejackreed) [inline app/models/payment.rb]: We do some other type of translations using a hash. Is it possible to keep a similar pattern for that?

Translation:
https://github.com/sul-dlss/mylibrary/blob/master/config/application.rb#L41-L69

Used:
https://github.com/sul-dlss/mylibrary/blob/a4b72fe7221fe57331733a283f8f6dddb799cfee/app/helpers/application_helper.rb#L63

**Comment by jvine** [comment]
It looks like only these two translations were added? These were just 2 examples I happened to find, but there were several other changes we made in the translations. All the Paid reason translations should match the Payable reasons.

File: app/models/payment.rb (hunk 1 above)
Comment (tallenaz) [reply app/models/payment.rb]: Thanks for catching that. Done in https://github.com/sul-dlss/mylibrary/pull/411/commits/d3391b7630a140b8dea230e87ec3a11853688f68

File: app/models/payment.rb [hunk 2]
```diff
@@ -32,7 +32,7 @@ def key
   end
 
   def bill_description
-    record['billReasonDescription']
+    Mylibrary::Application.config.bill_reasons[record['billReasonDescription']] || record['billReasonDescription']
```
Comment (mejackreed) [inline app/models/payment.rb]: Can we also move this to a helper, so other instances could access this native value also?

**Comment by tallenaz** [comment]
@jvine @saseestone the previous translation map was translating (we think) the fine status. The `bilReasonDescription` we're using in the "Reason" column from Symphony seems different from the fine status values. Is there a pre-existing mapping we can use for `billReasonDescription`?

**Comment by saseestone** [comment]
@tallenaz  it looks to me like in the fines tab, for Payable bills, we're using translations for "Reason" as outlined in #155.

But for Payment History (hopefully soon to become labelled just "History"), we're using the bilReasonDescription from the Symphony policy list for "Bill Reason".   Does that jive with what you're seeing in the code?

We should not be using the Symphony policy description in either case.  We should be using the translations as outlined in #155 .  Doable?

**Comment by tallenaz** [comment]
@saseestone this now uses a mapping from #155 instead of `billReasonDescription`.

**Pull request merged**